ecommerce/static/images/logo_primary.svg

```diff
@@ -1,329 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I couldn't comment on the above, but all these custom fonts and images seem to be missing from the theme repo.

Reply:
@UmanShahzad, I have added them in the theme repository now.